transport-native-kqueue/src/main/c/netty_kqueue_native.c

```diff
@@ -259,14 +260,62 @@ static jshort netty_kqueue_native_noteDisconnected(JNIEnv* env, jclass clazz) {
    return NOTE_DISCONNECTED;
 }
 
-static jint netty_kqueue_bsdsocket_connectResumeOnReadWrite(JNIEnv *env) {
+static jint netty_kqueue_bsdsocket_connectResumeOnReadWrite(JNIEnv *env, jclass clazz) {
     return CONNECT_RESUME_ON_READ_WRITE;
 }
 
-static jint netty_kqueue_bsdsocket_connectDataIdempotent(JNIEnv *env) {
+static jint netty_kqueue_bsdsocket_connectDataIdempotent(JNIEnv *env, jclass clazz) {
     return CONNECT_DATA_IDEMPOTENT;
 }
 
+static jint netty_kqueue_bsdsocket_fastOpenClient(JNIEnv *env, jclass clazz) {
+#if __APPLE__
```

Comment:
nit: usually we use `#ifdef` when possible
```suggestion
#ifdef __APPLE__
```